app/assets/javascripts/modules/doc-upload.js

```diff
@@ -35,18 +35,29 @@ moj.Modules.docUpload = {
       maxFilesize: parseInt(self.$form.data('max-filesize')),
       acceptedFiles: self.$form.data('accepted-files'),
       autoProcessQueue: true,
-      addRemoveLinks: true,
+      addRemoveLinks: false,
       createImageThumbnails: false,
       previewTemplate: previewTemplate,
-      dictRemoveFile: 'Remove',
       uploadMultiple: false,
       forceFallback: false,
       headers: {'X-CSRF-Token': $('meta[name="csrf-token"]').attr('content')},
 
+      dictInvalidFileType: ' is in a format we don\'t accept. You can upload JPG, PNG, GIF, Word or Excel files.',
```

Comment:
PDFs, too. 

We have a list of allowed filetypes in the translation file. Can we reuse it here?

          <li>File formats: Word, PDF, Excel, JPG, GIF, PNG</li>